components/status/StatusActionsMore.vue

```diff
@@ -5,6 +5,8 @@ const props = defineProps<{
   status: mastodon.v1.Status
   details?: boolean
   command?: boolean
```

Comment:
Adding this because I just noticed that it's possible to throw a fatal error when someone tries to manually fetch a status that has not federated to their instance.

Reply:
@patak-dev this PR is ready for re-review. This is only *new* change that I made, which is a bug fix to handle the scenario where the linked status has not federated to the instance where the post with the embedded link was posted. In this scenario, the dropdown previously included the option to "Copy link to this post" but when that link was subsequently entered into a browser's address bar, the user would hit an error page because the status `id` would be incorrect or missing altogether.

Here's an example of the fixed version using the preview environment: 
https://deploy-preview-2125--elk-zone.netlify.app/calckey.social/notes/9f6y642l66